src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1853,6 +1853,9 @@ bool CanDeleteSecretKey()
             }
 
             pullReplicationAsSink = JsonDeserializationClient.PullReplicationAsSink(pullReplicationBlittable);
+            
+            if (string.IsNullOrEmpty(pullReplicationAsSink.Url))
+                pullReplicationAsSink.Url = Configuration.Core.PublicServerUrl?.UriValue ?? Server.WebUrl;
```

Comment:
why we use ourself url, don't we need to use url from connection string?

Reply:
revert changes